docs/reliability/priming.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 8
+---
+
+# 🟡 Priming and Macros
+When interacting with an LLM (such as ChatGPT, which we’ll use as the baseline example for this article), your first input is weighted heavily as a guide for the rest of the conversation. As such, it’s possible to take advantage of that additional importance the model places on your input by setting up guidelines for your interaction in what’s referred to as a “priming prompt”. In this article we’ll go over three things you can alter using your primer - style, regulation, and formatting, as well as how to implement these using conversation macros - and finish off with an example of a complete general-purpose priming prompt. 
```

Comment:
Can you cite smtg for the idea that "first input is weighted heavily"

Reply:
Yep, will do